tests/loggers/test_mlflow_logger.py

```diff
@@ -16,6 +16,7 @@
 from composer.core import Callback, State
 from composer.loggers import Logger, MLFlowLogger
 from composer.trainer import Trainer
+from tests.callbacks.test_callbacks import clean_mlflow_runs
```

Comment:
you don't need to import fixtures. instead you need to put your fixture in fixtures.py and then the test machinery will make sure it is accessible as a fixture.